kalite/settings.py

```diff
@@ -169,7 +169,9 @@ def package_selected(package_name):
 
 import_installed_app_settings(INSTALLED_APPS, globals())
 
-KHAN_EXERCISES_DIRPATH = getattr(local_settings, "KHAN_EXERCISES_DIRPATH", os.path.join(os.path.dirname(__file__), "static", "perseus", "ke"))
+KHAN_EXERCISES_RELPATH = os.path.join("static", "perseus", "ke")
```

Comment:
looks like this is not meant to be replaced (no `getattr` pattern), so we should just put this on `i18n/__init__.py` instead.